geoportal/c2cgeoportal_geoportal/views/layers.py

```diff
@@ -66,10 +67,123 @@
 
 if TYPE_CHECKING:
     from c2cgeoportal_commons.models import main  # pylint: disable=ungrouped-imports.useless-suppression
+
+
 LOG = logging.getLogger(__name__)
 CACHE_REGION = get_region("std")
 
 
+class _LogLastUpdate:
```

Comment:
IMHO there is no need for a class here, a function should be enough.

Or you should call it `_BaseCallback` and put `_get_base_query` as a method of this class.